ai-machine-learning/thoth-thamos/advise.md

```diff
@@ -0,0 +1,49 @@
+Thamos CLI runs based on the `.thoth.yaml` config, which we have setup already in the example repository here. 
+
+You can browse the config file in the editor or via terminal using -
+
+``cat .thoth.yaml``{{execute}} 
+
+Now we are ready to do ask Thoth for Advise -
+
+``thamos advise``{{execute}}
+
+Tip - ``thamos advise --force`` runs the CLI without cache.
+
+Be a little patient here, this takes some time. After this is done you should see - 
```

Comment:
Can you add a `<br>` at the end of this line, so the image always shows below the text?
Same for any other images